application/src/main/java/org/togetherjava/tjbot/util/TriConsumer.java

```diff
@@ -0,0 +1,24 @@
+package org.togetherjava.tjbot.util;
+
+/**
+ * Extension of {@link java.util.function.BiConsumer} but for 3 elements.
+ * <p>
+ * Represents an operation that accepts three input arguments and returns no result. This is the
+ * three-arity specialization of {@link java.util.function.Consumer}. Unlike most other functional
+ * interfaces, TriConsumer is expected to operate via side effects.
+ *
+ * @param <A> the type of the first argument to the operation
+ * @param <B> the type of the second argument to the operation
+ * @param <C> the type of the third argument to the operation
+ */
+@FunctionalInterface
+public interface TriConsumer<A, B, C> {
```

Comment:
If this is only used in the CommandSystem, why not give it a better name, remove the generics and inline it in the class?

Reply:
Possible. Thought it might be useful for others too. Its a pretty self explanatory and standard interface, I would suppose. But sure, I can inline it.